.github/ISSUE_TEMPLATE/0-turborepo-bug-report.yml

```diff
@@ -39,6 +39,7 @@ body:
         - pnpm
         - Yarn v1
         - Yarn v2/v3 (node_modules linker only)
+        - Yarn v4
```

Comment:
I think we should bucket this with the previous line since the linker mode still matters (v4 still uses pnp by default)

Reply:
oh wow, that is cool, TIL! Seems better in theory than node_modules linker